e2e/_suites/fleet/installers.go

```diff
@@ -214,25 +215,40 @@ func (i *TARPackage) Preinstall() error {
 		return err
 	}
 
-	version := checkElasticAgentVersion(i.version)
+	version := i.version
+	if !i.stale {
+		version = checkElasticAgentVersion(i.version)
+	}
 
 	// simplify layout
-	cmds := []string{"mv", fmt.Sprintf("/%s-%s-%s-%s", i.artifact, version, i.OS, i.arch), "/elastic-agent"}
-	err = execCommandInService(i.profile, i.image, i.service, cmds, false)
-	if err != nil {
-		log.WithFields(log.Fields{
-			"command": cmds,
-			"error":   err,
-			"image":   i.image,
-			"service": i.service,
-		}).Error("Could not extract agent package in the box")
-
-		return err
+	cmds := [][]string{
+		[]string{"rm", "-fr", "/elastic-agent"},
```

Comment:
When running all Fleet tests in my local machine, I saw multiple errors when moving the agent, because `mv` will refuse to rename a directory to another directory if the target directory contains files.

It does not happen on CI but in my local machine happens after multiple runs. I'm adding it defensively, as it's harmless.